corehq/apps/app_manager/suite_xml.py

```diff
@@ -920,8 +926,63 @@ def details(self):
 
                         if detail_column_infos:
                             if detail.custom_xml:
-                                d = load_xmlobject_from_string(detail.custom_xml, xmlclass=Detail)
+                                d = load_xmlobject_from_string(
+                                    detail.custom_xml,
+                                    xmlclass=Detail
+                                )
                                 r.append(d)
+
+                            elif detail.use_case_tiles:
```

Comment:
It would be nice to refactor this block into a separate function. As is it's very deeply nested and makes the function pretty long, which makes a pretty bad smell.